ex03.ex

```diff
@@ -59,7 +59,13 @@ defmodule Ex03 do
 
   """
 
-  def odd_even . . . "your code"
+  def odd?(true), do: :odd
```

Comment:
# +1

for some great naming. 

One small thing: normally the `?` suffix is reserved for predicate functions (that return `true` or `false`)